components/lsp/MintForm.tsx

```diff
@@ -73,6 +76,17 @@ const MintForm: FC<Props> = ({
     }
   }, [signer, setShowWallet]);
 
+  useEffect(() => {
+    if (
+      amount &&
+      amount > ethers.utils.formatUnits(collateralBalance, collateralDecimals)
```

Comment:
I think we should be converting these to some sort of numerical representation (BigNumber/BN probably) before comparing rather than comparing strings.

Reply:
agreed. the biggest problem im seeing is incorrect number comparison since its being compared alphabetically. IE i have 20 uma and type 9, and it says i dont have enough